packages/core/src/lib/actions/session.ts

```diff
@@ -126,6 +126,7 @@ export async function session(
         // By default, only exposes a limited subset of information to the client
         // as needed for presentation purposes (e.g. "you are logged in as...").
         session: {
+          id: session.id,
```

Comment:
Honestly, I think the whole `adapterSession` should be available in the `session()`-callback

Reply:
This is not desired. This callback controls what gets exposed to the browser. We don't want to expose anything than a strict subset. session id should not be needed in the client.

Reply:
I kinda want to decide what's on my session object myself based on what I have in the db, but I get your point. Is session is alright?